src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -152,10 +152,13 @@ private static void ShowResults(UpdateResults results)
 					}
 				}
 
-				Trace.WriteLine($"{results.PropsFileUpdateResults.UnrecognizedVariables.Count} variables were not found in props files:");
-				foreach (var variable in results.PropsFileUpdateResults.UnrecognizedVariables)
+				if(results.PropsFileUpdateResults.UnrecognizedVariables.Count != 0)

```

Comment:
very minor: please check the rest of the code base for homogeneity in this case...
`.Any()` vs `.Count !=0`

my recommendation would be to strive for style homogeneity.

Reply:
In all the cases I found in the code, we use Count except when the count of the collection is not known (IEnumerable<>).